File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/04.For-Loop/02-exercise/05.salary.js
function salary(input) {

    const fineForFacebook = 150;
    const fineForInstagram = 100;
    const fineForReddit = 50;

    let index = 0;
    let openTabsCount = Number(input[index]);
    index++;
    let salary = Number(input[index]);
    index++;

    let isHaveSalary = true;

    for (let i = 1; i <= openTabsCount; i++) {
        let currentTab = input[index];
        index++;

        switch (currentTab) {
            case "Facebook": salary -= fineForFacebook; break;
            case "Instagram": salary -= fineForInstagram; break;
            case "Reddit": salary -= fineForReddit; break;
        }

        if (salary <= 0) {
            console.log("You have lost your salary.");
            isHaveSalary = false;
            break;
        }
    }

    if (isHaveSalary) {
        console.log(Math.trunc(salary));
    }
}
salary(["3", "500", "Facebook", "Dev.bg", "SoftUni"]);

File: Python_Basics/Homeworks-And-Labs/04.for_loop/02.exercise/05.salary.py
fine_for_facebook = 150
fine_for_instagram = 100
fine_for_reddit = 50

open_tabs_count = int(input())
salary = float(input())

is_have_salary = True

for i in range(open_tabs_count):
    current_tab = input()

    if current_tab == 'Facebook':
        salary -= fine_for_facebook
    elif current_tab == 'Instagram':
        salary -= fine_for_instagram
    elif current_tab == 'Reddit':
        salary -= fine_for_reddit

    if salary <= 0:
        print("You have lost your salary.")
        is_have_salary = False
        break

if is_have_salary:
    print(int(salary))
